rfcs/xxxx-v5-content-api.md

```diff
@@ -0,0 +1,217 @@
+- Start Date: 2023-09-05
+- RFC PR: (leave this empty)
+
+# Summary
+
+V5 is in preparation and will require making changes to the Content APIs (REST & GraphQL).
+
+# Motivation
+
+V5 Content Engine requires some breaking changes to the API, It is also the opportunity to simplify the API Response format following user feedbacks.
+
+> The main goal of this RFC is to propose a simplified and v5 compatible format while offering a smoother migration path with a legacy format.
+
+# Detailed design
+
+### Endpoints
+
+Based on the [Database changes planned](https://github.com/strapi/rfcs/pull/52),we will now mainly work with the `documentId` within the API.
+
+**Collection Types**
+
+| Method   | Url                                             | Desc                                                                    |
+| -------- | ----------------------------------------------- | ----------------------------------------------------------------------- |
+| `GET`    | `/api/:contentType`                             | Find a list of documents                                                |
```

Comment:
This feels like filtering for pivot states will be baked into filters api. I think it would benefit, if there would be clear GET endpoints for like "all french documents" or "all live documents".